src/Sulu/Bundle/MediaBundle/DependencyInjection/SuluMediaExtension.php

```diff
@@ -54,6 +54,7 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setParameter('sulu_media.format_cache.path', '%kernel.root_dir%/../' . $config['format_cache']['public_folder'] . '/uploads/media');
         $container->setParameter('sulu_media.format_cache.segments', '10');
         $container->setParameter('ghost_script.path', $config['ghost_script']['path']);
+        $container->setParameter('sulu_media.image.formats', array()); // written by compiler pass
```

Comment:
@alexander-schranz I wouldn't put this line here, since it is written by the compiler pass. @dantleech WDYT?